pkg/tekton/tekton.go

```diff
@@ -0,0 +1,45 @@
+package tekton
+
+import (
+	"github.com/codeready-toolchain/toolchain-operator/pkg/apis/toolchain/v1alpha1"
+	olmv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	SubscriptionNamespace = "openshift-operators"
+	SubscriptionName      = "openshift-pipelines-operator"
+	SubscriptionSuccess   = "tekton operator subscription created"
+)
+
+//NewSubscription for openshift-pipeline operator
+func NewSubscription(ns string) *olmv1alpha1.Subscription {
+	return &olmv1alpha1.Subscription{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      SubscriptionName,
+			Namespace: ns,
+			Labels:    Labels(),
+		},
+		Spec: &olmv1alpha1.SubscriptionSpec{
+			Channel:                "dev-preview",
+			Package:                "openshift-pipelines-operator",
+			StartingCSV:            "openshift-pipelines-operator.v0.5.2",
+			CatalogSource:          "community-operators",
+			CatalogSourceNamespace: "openshift-marketplace",
+		},
+	}
+}
+
+func Labels() map[string]string {
```

Comment:
We probably can reuse the same `Label()` function for all our resources (Che & Tekton subscription for now). Not sure where we can move `Label()` function to though...